EIPS/eip-micropayments_standard_for_nfts_and_multi_tokens.md

```diff
@@ -0,0 +1,329 @@
+---
+eip: 4393
+title: Micropayments Standard for NFTs and Multi Tokens
+description: A standard interface for tip tokens that allows tipping to holders of NFTs and multi tokens
+author: Jules Lai (@julesl23)
+discussions-to: https://ethereum-magicians.org/t/eip-proposal-micropayments-standard-for-nfts-and-multi-tokens/7366
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-10-24
+requires: 165, 721, 1155
+---
+
+## Abstract
```

Comment:
The abstract should be a short, technical description of the EIP. Please update this before moving forward into review.

Reply:
Thanks. Updated.